inspectit-oce-bootstrap/src/main/java/rocks/inspectit/oce/bootstrap/noop/NoopContext.java

```diff
@@ -0,0 +1,34 @@
+package rocks.inspectit.oce.bootstrap.noop;
```

Comment:
let's match the original package and noop?

* `rocks.inspectit.oce.bootstrap.context` -> `rocks.inspectit.oce.bootstrap.context.noop`
* `rocks.inspectit.oce.bootstrap.instrumentation` -> `rocks.inspectit.oce.bootstrap.instrumentation.noop`